boards/msbiot/Makefile.include

```diff
@@ -14,3 +14,8 @@ export STLINK_VERSION ?= 2
 
 # this board uses openocd
 include $(RIOTMAKE)/tools/openocd.inc.mk
+
+# add driver for CC1101 sub-gigahertz transceiver as default netdev
+ifneq (,$(filter netdev_default gnrc_netdev_default,$(USEMODULE)))
+	USEMODULE += cc110x
```

Comment:
indent should be only 2 spaces

Reply:
Thanks for the hint. I will fix this. Is there a reason why RIOT uses 4 spaces in C code, but 2 in Makefiles? This seems inconsistent.

I personally use smart tabs [1](https://www.stream9.net/blog/post/spaces-or-tabs-the-answer-is-both), [2](https://www.emacswiki.org/emacs/SmartTabs) for just this reason: You don't need to argue about indent width, because it because an editor option everyone can configure to her/his liking. And if needed, to different width on a per file type bases.